doc/coding-conventions.xml

```diff
@@ -254,10 +254,13 @@ bound to the variable name <varname>e2fsprogs</varname> in
   dash) — e.g., <literal>"hello-0.3.1rc2"</literal>.</para></listitem>
 
   <listitem><para>If a package is not a release but a commit from a repository, then
-  the version part of the name <emphasis>must</emphasis> be the date of that 
+  the version part of the name should be the date of that 
```

Comment:
Should this be `SHOULD` and the next line be `MUST`, for clarity's sake?
 ref: https://www.ietf.org/rfc/rfc2119.txt.